<launch>
    <arg name="kp" default="0.025"/>
    <rosparam  param="kp" subst_value="True">$(arg kp)</rosparam>




    <node pkg="nhk_2020" type="Run" name="Run" output="screen">
    </node>
    <node pkg="nhk_2020" type="Omni" name="Omni" output="screen">
    </node>
    <node pkg="nhk_2020" type="action" name="action" output="screen">
    </node>
    <node pkg="nhk_2020" type="Blue.py" name="StateMachine" output="screen">
    </node>
    
    <include file ="$(find nhk_2020)/launch/arduino.launch"/>
</launch>
